platform/android/export/export.cpp

```diff
@@ -2234,9 +2260,27 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 	Error sign_apk(const Ref<EditorExportPreset> &p_preset, bool p_debug, const String &export_path, EditorProgress &ep) {
 		int export_format = int(p_preset->get("custom_template/export_format"));
 		String export_label = export_format == EXPORT_FORMAT_AAB ? "AAB" : "APK";
-		String release_keystore = p_preset->get("keystore/release");
-		String release_username = p_preset->get("keystore/release_user");
-		String release_password = p_preset->get("keystore/release_password");
+
+		String release_keystore;
+		if (OS::get_singleton()->has_environment(ENV_KEYSTORE_PATH)) {
+			release_keystore = OS::get_singleton()->get_environment(ENV_KEYSTORE_PATH);
+		} else {
+			release_keystore = p_preset->get("keystore/release");
+		}
+
+		String release_username;
+		if (OS::get_singleton()->has_environment(ENV_KEYSTORE_USER)) {
+			release_username = OS::get_singleton()->get_environment(ENV_KEYSTORE_USER);
+		} else {
+			release_username = p_preset->get("keystore/release_user");
+		}
+
+		String release_password;
+		if (OS::get_singleton()->has_environment(ENV_KEYSTORE_PASSWORD)) {
+			release_password = OS::get_singleton()->get_environment(ENV_KEYSTORE_PASSWORD);
+		} else {
+			release_password = p_preset->get("keystore/release_password");
+		}
```

Comment:
We are repeating similar logics here, above and below. Can the retrieval logic be wrapped into a helper function instead. This would ensure that when we update on section, we don't introduce bugs by omitting to update the other sections.
The helper function could return a `struct` containing the `keystore_path`, `keystore_username` and `keystore_password`, as well as an error indicating if the returned information is valid or not.